plugins/tech-radar/src/components/RadarDescription/RadarDescription.tsx

```diff
@@ -48,6 +62,62 @@ const RadarDescription = (props: Props): JSX.Element => {
       </DialogTitle>
       <DialogContent dividers>
         <MarkdownContent content={description} />
+        <Typography variant="h6" gutterBottom>
```

Comment:
I wonder if we should extract this into a timeline component instead of polluting the whole description component?

Reply:
Yes we can do that. There as not much in the Description component to begin with, but for future changes and extra functionality it would make sense to move the timeline to a Timeline component already.